flaeg.go

```diff
@@ -549,9 +437,13 @@ func LoadWithCommand(cmd *Command, cmdArgs []string, customParsers map[reflect.T
 		return err
 	}
 
-	valmap, err := parseArgs(cmdArgs, tagsmap, parsers)
-	if err != nil {
-		return PrintErrorWithCommand(err, tagsmap, defaultValmap, parsers, cmd, subCommand)
+	valmap, errParseArgs := parseArgs(cmdArgs, tagsmap, parsers)
+	if errParseArgs != nil {
+		if errParseArgs == ErrParserNotFound {
+			fmt.Println(errParseArgs.Error())
```

Comment:
There is a problem here :cry: 
We should return the error instead of loging it.
In staert, we use flaeg without custom parser as we do there : 
https://github.com/containous/staert/blob/master/staert.go#L156